Libraries/CMSIS/Device/Maxim/MAX32670/Source/ARM/startup_max32670.s

```diff
@@ -297,7 +298,7 @@ Default_Handler\
                 EXPORT  RSV21_IRQHandler              [WEAK]  ; 0x25  0x0094  37: Reserved
                 EXPORT  RSV22_IRQHandler              [WEAK]  ; 0x26  0x0098  38: Reserved
                 EXPORT  FLC0_IRQHandler               [WEAK]  ; 0x27  0x009C  39: Flash Controller 0
-                EXPORT  GPIO0_IRQHandler              [WEAK]  ; 0x28  0x00A0  40: GPIO0
+                ;EXPORT  GPIO0_IRQHandler              [WEAK]  ; 0x28  0x00A0  40: GPIO0
```

Comment:
I don't see this type of change in the GCC or IAR code.  If the line does need to be commented out, it may be worth replacing it with a comment that explains why it is being skipped in this table.

Reply:
Added comment to justify commenting out the Export because of the presence of a weak function in board.c